instance/migrations/0063_auto_20161104_1720.py

```diff
@@ -0,0 +1,54 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.9.5 on 2016-11-04 17:20
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+import django.db.models.deletion
+import django_extensions.db.fields
+import instance.models.utils
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('instance', '0062_load_balancer'),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='RabbitMQUser',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('created', django_extensions.db.fields.CreationDateTimeField(auto_now_add=True, verbose_name='created')),
+                ('modified', django_extensions.db.fields.ModificationDateTimeField(auto_now=True, verbose_name='modified')),
+                ('username', models.CharField(max_length=32, unique=True)),
+                ('password', models.CharField(max_length=64)),
+            ],
+            options={
+                'ordering': ('-modified', '-created'),
+                'get_latest_by': 'modified',
+                'abstract': False,
+            },
+            bases=(instance.models.utils.ValidateModelMixin, models.Model),
+        ),
+        migrations.AddField(
+            model_name='openedxinstance',
+            name='rabbitmq_provisioned',
+            field=models.BooleanField(default=False),
+        ),
+        migrations.AddField(
```

Comment:
Hmm, I don't think so - we're going to roll this out to all instances immediately that are not using the "ephemeral databases" option, and I don't think there's a use case for keeping any non-ephemeral instances using the local RabbitMQ.

Also, these models are used for adding vhosts/users to the RabbitMQ setup configured in OCIM, so we'd need to have some extra checks/another boolean field in here if we wanted to support using local RabbitMQ setups on non-ephemeral instances.